ib_manifest_util/create_hardening_manifest.py

```diff
@@ -158,6 +159,14 @@ def update_hardening_manifest(
         )
     hardening_data["tags"] = [dockerfile_version]
 
+    # store resources with underscores (we'll need to flag them later)
+    unclean_resources = deepcopy(hardening_data["resources"])
+
+    # clean the leading underscores for the hardening_manifest
+    for idx, resource in enumerate(hardening_data["resources"]):
+        if resource["filename"][0] == "_":
+            hardening_data["resources"][idx]["filename"] = resource["filename"][1:]
+
```

Comment:
This perhaps more of a style choice but `if resource["filename"].startswith("_")` and `... = resource["filename"].lstrip("_")` works well too :)

Reply:
that's actually much more resilient. I love it. Thanks! :)